components/match2/wikis/warcraft/match_summary.lua

```diff
@@ -0,0 +1,347 @@
+---
+-- @Liquipedia
+-- wiki=warcraft
+-- page=Module:MatchSummary
+--
+-- Please see https://github.com/Liquipedia/Lua-Modules to contribute
+--
+
+local Array = require('Module:Array')
+local DisplayHelper = require('Module:MatchGroup/Display/Helper')
+local DisplayUtil = require('Module:DisplayUtil')
+local Lua = require('Module:Lua')
+local WarcraftMatchExternalLinks = require('Module:MatchExternalLinks/Warcraft')
+
+local MatchGroupUtil = Lua.import('Module:MatchGroup/Util', {requireDevIfEnabled = true})
+local WarcraftMatchGroupUtil = Lua.import('Module:MatchGroup/Util/Warcraft', {requireDevIfEnabled = true})
+local WarcraftOpponentDisplay = Lua.import('Module:OpponentDisplay/Warcraft', {requireDevIfEnabled = true})
+local RaceIcon = Lua.requireIfExists('Module:RaceIcon') or {
+	getTinyIcon = function() end,
+}

```

Comment:
currently not used in this module
so either some stuff is missing (where it will be used) or these lines can be removed

Reply:
This will be needed eventually to make the icons for the match summary.  Can remove for a minimum viable product or stick around for when it is used.